lib/Network/OAuth2/Server/Statistics.hs

```diff
@@ -24,7 +33,7 @@ import           Network.OAuth2.Server.Types
 statsLogName :: String
 statsLogName = "Tokens.Server.Statistics"
 
-
+-- | Counters for EKG monitoring
```

Comment:
"The counters are not added to the EKG server state but will be collected on demand by gatherStats."